cmd/proxy-to-grafana/proxy-to-grafana.go

```diff
@@ -0,0 +1,129 @@
+// Copyright (c) 2022 Tailscale Inc & AUTHORS All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// proxy-to-grafana uses a reverse proxy which identifies users based on their
+// originating Tailscale node and creates / logs in users to grafana. Uses the
```

Comment:
Grafana capitalized

Reply:
It uses

It uses Grafana's AuthProxy

("Grafana AuthProxy" is how they seem to mostly spell it in their docs)

Reply:
👍